mailpoet/tasks/code_sniffer/composer.json

```diff
@@ -8,7 +8,7 @@
     "slevomat/coding-standard": "^8.15",
     "squizlabs/php_codesniffer": "^3.9",
     "wporg/plugin-directory": "dev-trunk",
-    "wp-coding-standards/wpcs": "^3.0"
+    "wp-coding-standards/wpcs": "^2.0"
```

Comment:
Did you have to downgrade here for some reason?

Reply:
Yes there was an incompatibility with the `wporg/plugin-directory` package where we take the "minimum" plugin rules. I think we may want to replace that and upgrade at some point but for the sake of getting this passing I downgraded it.